lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -1634,49 +1607,47 @@ class MM600SeriesDriver extends BCNLDriver {
     };
   }
 
-  enterRemoteCommandMode() {
-    return new BCNLCommand(BCNLMessage.ASCII_CONTROL.NAK).send(this.hidDevice)
-      .then(response => response.checkAsciiControl(BCNLMessage.ASCII_CONTROL.EOT))
-      .then(() => new BCNLCommand(BCNLMessage.ASCII_CONTROL.ENQ).send(this.hidDevice))
-      .then(response => response.checkAsciiControl(BCNLMessage.ASCII_CONTROL.ACK));
+  async enterRemoteCommandMode() {
```

Comment:
I can't get over how nice and readable the async/await code looks compared to the "old" promises. :+1:

Reply:
Yeah. They're really nice to work with 😄